### PR TITLE
Feature/ballot nav

### PR DESCRIPTION
**Pull request by mford008**
[WIP] To address issue #152 in Launch plan (Temporarily hide ballot measures from nav).

Still working on having this active in admin view only.

### CODE REVIEW

**Comment by adborden** [comment]
:+1: can you also update [robots.txt](https://github.com/caciviclab/odca-jekyll/blob/feature/alert-markdown/robots.txt)? Maybe just disallow the entire site, it's easier that way.

File: _includes/ballot-nav.html [hunk 1]
```diff
@@ -4,17 +4,20 @@
...
+        <div class="l-section__content">
+          <h3>{% include referendum_measure_heading.html referendum_measure_display=locality.referendum_measure_display %}</h3>
+          {% for ballot_item in referendums %}
+            {% assign referendum = site.referendums | where: "path", ballot_item | first %}
+            <p><a href="{{ referendum.url | prepend: site.baseurl }}">{% if referendum.number %}{{ referendum.number }} {% endif %}{{ referendum.title | escape}}</a></p>
+          {% endfor %}
+        </div>
+      </section>
+    {% endif %}
+  {% endcomment %}
```
Comment (adborden) [inline _includes/ballot-nav.html]: You can just delete this, git will allow us to bring it back.

Comment (mford008) [reply _includes/ballot-nav.html]: Will do!

Comment (mford008) [reply _includes/ballot-nav.html]: So just to be clear, you want robots.txt line 2 to say "Disallow: /site/"?

**Comment by adborden** [comment]
On Wed, 2018-07-25 at 20:40 -0700, Maddy Ford <notifications@github.com> wrote:
> So just to be clear, you want robots.txt line 2 to say "Disallow: /site/"? 

It should be:

Disallow: /

To disable the entire site.

--
Aaron D Borden
Human and Hacker

**Comment by ckingbailey** [comment]
I added Disallow: / to robots.txt

**Comment by adborden** [comment]
Looking forward to it!

**Comment by ckingbailey** [comment]
Hey @adborden, tests are not passing on this branch since our changes and @mford008 and I are not sure how to proceed.
Not sure why tests are not passing. Neither of use understand the error message we're getting. (Line 297 of which file? I'm not sure how to hunt down the broken internal link that is producing the error.)
screen shot 2018-07-31 at 8 32 47 pm

**Comment by ckingbailey** [comment]
One thing I thought of is to revert to the last passing commit, 914600c, but I wasn't sure of the best way to do that.

**Comment by adborden** [comment]
@ckingbailey don't worry about the failing tests. One downside to how we run our builds is that we pull from the latest data. This means that in some cases our builds will start failing because of some external change (new finance data). In this case, the test is saying there is a broken link because a committee exists in the data, but we don't have a filer_id for it. To fix, we usually just have to add the filer_id to the Google Spreadsheet. I'll dig into it soon and get this merged.

**Comment by adborden** [comment]
BTW, the error is line 297 of `_site/referendum/sf/2018-06-05/parcel-tax-for-san-francisco-unified-school-district/index.html`. `_site` is the default Jekyll build directory. This is where all the compiled HTML, CSS, JS, ends up. One of the tests just runs some common checks on those output files, checking for broken links, invalid HTML, duplicate id attributes, missing alt attributes, etc.

**Comment by adborden** [comment]
@mford008 looks like you're mixing in some work for another issue into this PR, adding the totals to the landing page. Do you mind removing it?

**Comment by ckingbailey** [comment]
ah, ok, cool, thanks @adborden

**Comment by ckingbailey** [comment]
I reverted the "adding oakland total to landing page" commit. Is this ready to merge?
** I cherry-picked the reverted commit onto the branch feature/totals

**Comment by adborden** [comment]
Close, there's still some extra CSS changes in there. I just [rebased](https://github.com/caciviclab/odca-jekyll/compare/feature/ballot-nav-2) this branch and dropped the unwanted commits so the history is cleaner. The missing committees are in the spreadsheet, so they should get picked up in the build tomorrow. At that point I'll push the rebased branch here and merge it.

@mford008 I think you might be in a bit of a mess once I push this. I can help you untangle everything on Tuesday.

**Pull request merged**